### ISSUE TITLE
[hiccup/hdom]: boolean attributes are not expressed in html as 

### ISSUE THREAD

**Issue by loganpowell** [thi-ng/umbrella#133]
in the hiccup docs I see this example:

```js
serialize(
    ["div.notice",
        {
            selected: true,
            style: {
                background: "#ff0",
                border: "3px solid black"
            }
        },
        "WARNING"]
);
```
```js
<div class="notice" selected style="background:#ff0;border:3px solid black">WARNING</div>
```
but when I use the `selected: true` syntax I end up with:
```js
<div class="notice" selected="true" style="background:#ff0;border:3px solid black">WARNING</div>
```
I am using `hiccup` via `hdom` if that is the reason...

**Comment by loganpowell** [comment]
Here's the example:
https://loganpowell.github.io/thi.ng_learning/dist/

**Comment by postspectacular** [comment]
Hi @loganpowell - this is a regression & thanks for finding it! I already have a fix ready (incl. support for more element properties). Btw. when you're using hdom, you're not actually using hiccup as underlying implementation (only some config/constants from the latter). Both packages just use the same syntax and (in theory) should be fully compatible...

**Comment by loganpowell** [comment]
Good to know!

**Comment by postspectacular** [comment]
fixed in hdom v8.0.6 (just released)
